### PR TITLE
i18n: Toggle tippy content on lang change

### LINKED ISSUE #87: "Active" explanation in Japanese
We've added the explanation of Active as a tooltip, but it's a bit tricky to swap the languages since it is done using an attribute. I think we'll either have to update the element's tippy attribute directly on language switch, or use a tippy API endpoint to manually set the text on language switch.

image

### PR DESCRIPTION
**Pull request by colinthornton**
@reustle @liquidx 
Resolves #87

## Updates
- Adds `kpi-active-tooltip`  translations
- Toggles tippy translations via the `data-tippy-i18n` attribute (can be extended to other elements later if more tooltips become necessary)
- Fixes a bug where last updated time doesn't update on language toggle

### CODE REVIEW

**Comment by reustle** [comment]
Thank you! I agree with waiting until #124 is merged

**Comment by colinthornton** [comment]
Cool, I'll update this code to follow whatever guidelines that PR sets once it's been merged.

**Comment by reustle** [comment]
#124 is now merged 🎉

**Comment by colinthornton** [comment]
I'll take a look today after work!

**Comment by mertd** [comment]
I documented how to use the i18n framework here: https://github.com/reustle/covid19japan#localization

**Comment by colinthornton** [comment]
@reustle @liquidx 
This should be good to go.

I also included a fix for the last updated time translations.

File: src/index.js [hunk 1]
```diff
@@ -967,8 +968,19 @@ function drawLastUpdated(lastUpdated) {
 
   display.textContent = relativeTime[LANG];
   display.setAttribute("title", lastUpdated);
-  display.setAttribute("data-en", relativeTime["en"]);
-  display.setAttribute("data-ja", relativeTime["ja"]);
+  i18next.addResource(
```
Comment (liquidx) [inline src/index.js:971]: I tried to fix the same thing by calling drawLastUpdated() in setLang(), but this is a better fix than mine.

Could you merge to the latest, and remove the call to drawLastUpdated() in setLang()? This isn't required any more.

File: src/index.js [hunk 2]
```diff
@@ -1143,12 +1155,28 @@ function setLang(lng) {
       if (document.getElementById("travel-restrictions")) {
         drawTravelRestrictions();
       }
-      drawLastUpdated(ddb.lastUpdated);
```
Comment (colinthornton) [inline src/index.js:1146]: @liquidx Done!

**Comment by mertd** [comment]
Something seems to be off, at least on the [netflify deploy preview](https://deploy-preview-144--covid19japan.netlify.com/):
1. Right after accessing the page, all is fine: 
grafik
2. After changing to Japanese:
grafik
3. After changing back to English:
grafik

**Comment by colinthornton** [comment]
@mertd Thanks for spotting that, it looks like the update to LANG changed places when I pulled in master earlier. Fixed at https://github.com/reustle/covid19japan/pull/144/commits/6775391bc6b07312e3a8674fc0a98fa0e1954bb3

**Comment by colinthornton** [comment]
@reustle I pulled in the latest changes, this should be ready to go.

**Comment by liquidx** [comment]
Thanks for the PR! Merged.

**Pull request merged**